src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/AzureCognitiveServices.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Security.Utilities;
+
+public enum AzureCognitiveServices
```

Comment:
Non-flags enum should not be plural.

```suggestion
public enum AzureCognitiveService
``` #Resolved

Reply:
Thank you, FxCop engineer emeritus. I suppose you didn't comment on the absence of a zero `None` value because we are non-flags and, in fact, put the most common/default value in this position.